book/src/user/lightwalletd.md

```diff
@@ -0,0 +1,248 @@
+source .profile
+```
+
+With go installed and in your path, download and install lightwalletd:
+
+```console
+git clone https://github.com/adityapk00/lightwalletd
+cd lightwalletd
+make
+make install
+```
+
+If everything went good you should have a `lightwalletd` binary in the root of the project.
+
+## Sync lightwalled
+[#sync-lightwalletd]: (#sync-lightwalletd)
+
+Please make sure you have zebrad running(with RPC endpoint and up to date blockchain) to synchronize lightwalletd.
+
+```console
+./lightwalletd --no-tls-very-insecure --zcash-conf-path ~/.zcash/zcash.conf --data-dir . --log-file /dev/stdout
+```
+
+Lightwalletd expects a `zcash.conf` file with just the following content:
+
+```
+txindex=1
+insightexplorer=1
+experimentalfeatures=1
+```
+
+With the above command lightwalletd will create a database folder `db` in the root of the project (we told lightwalletd to do so in the `--data-dir .` part of the command).
+
+By default lightwalletd service will listen on `127.0.0.1:9067`
+
+Lightwalletd will do its own synchronization, while it is doing you will see messages as:
+
+```console
+...
+app":"lightwalletd","level":"info","msg":"Ingestor adding block to cache: 748000","time":"2022-05-28T19:25:49-03:00"}
+{"app":"lightwalletd","level":"info","msg":"Ingestor adding block to cache: 749540","time":"2022-05-28T19:25:53-03:00"}
+{"app":"lightwalletd","level":"info","msg":"Ingestor adding block to cache: 751074","time":"2022-05-28T19:25:57-03:00"}
+...
+```
+
+Wait until lightwalletd is in sync before connecting any wallet into it. You will know when it is in sync as those messages will not be displayed anymore.
+
+## Run tests
+[#run-tests]: (#run-tests)
+
+The Zebra team created tests for the interaction of zebra and lightwalletd. 
+
+Please refer to [acceptance](https://github.com/ZcashFoundation/zebra/blob/v1.0.0-beta.10/zebrad/tests/acceptance.rs) tests documentation in the `Lightwalletd tests` section.
+
+## Connect a wallet to lightwalletd
+[#connect-wallet-to-lightwalletd]: (#connect-wallet-to-lightwalletd)
+
+The final goal is to connect wallets to the lightwalletd service backed by Zebra. 
+
+For demo purposes we use [zecwallet-cli](https://github.com/adityapk00/zecwallet-light-cli).
+
+Make sure both `zebrad` and `lightwalletd` are running and listening.
+
+### Download and build the cli-wallet
+[#download-and-build-the-cli-wallet]: (#download-and-build-the-cli-wallet)
+
+```console
+$ git clone https://github.com/adityapk00/zecwallet-light-cli.git
+$ cd zecwallet-light-cli
+$ cargo build --release
+```
+
+### Run the wallet
+[#run-the-wallet]: (#run-the-wallet)
+
+```console
+$ ./target/release/zecwallet-cli --server 127.0.0.1:9067
+Lightclient connecting to http://127.0.0.1:9067/
+{
+  "result": "success",
+  "latest_block": 1683911,
+  "total_blocks_synced": 49476
+}
+Ready!
+(main) Block:1683911 (type 'help') >> 
+```
+## Lightwalletd and nginx
+[#lightwalletd-and-nginx]: (#lightwalletd-and-nginx)
```

Comment:
The `nginx` section seems like it isn't Zebra-specific.

Does it belong in the `lightwalletd` repository's documentation instead?

Reply:
Removed at https://github.com/ZcashFoundation/zebra/pull/4526/commits/a11be48cbefd0d5b05720fc8c4e117a7de33632d